src/zzz_od/application/shiyu_defense/shiyu_defense_team_utils.py

```
@@ -258,21 +256,102 @@ def calc_teams(
+        if not has_unfinished:
+            team_info = DefensePhaseTeamInfo([DmgTypeEnum.UNKNOWN, DmgTypeEnum.UNKNOWN],
+                                              [DmgTypeEnum.UNKNOWN, DmgTypeEnum.UNKNOWN])
+            team_info.team_idx = -1  # 标记跳过
+            team_list.append(team_info)
+            continue
+
+        # 属性识别：用"第一间属性"等区域
+        attr_name = f'{room_name}属性'
+        try:
+            area = ctx.screen_loader.get_area(screen_template, attr_name)
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)